canvas_modules/harness/src/client/components/custom-canvases/tables/tables-canvas.jsx

```diff
@@ -40,29 +40,38 @@ export default class TablesCanvas extends React.Component {
 			enableAssocLinkCreation: true,
 			enableAssocLinkType: "RightSideCurve",
 			enableParentClass: "tables-join",
+			enableHightlightNodeOnNewLinkDrag: true,
```

Comment:
Was there a typo from before when this was first added? Should this actually be `enableHighlightNodeOnNewLinkDrag` instead?

Reply:
OMG Yes, this is actually a type from when the option was first introduced. The name is so long I think I've always copy and pasted it instead of typing in from scratch so I never noticed. I've opened this issue to change it in future. 
https://github.com/elyra-ai/canvas/issues/503